xchange-coinmate/src/main/java/info/bitrich/xchange/coinmate/CoinmateStreamingAccountService.java

```diff
@@ -0,0 +1,68 @@
+package info.bitrich.xchange.coinmate;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import info.bitrich.xchange.coinmate.dto.CoinmateWebsocketBalance;
+import info.bitrich.xchangestream.core.StreamingAccountService;
+import info.bitrich.xchangestream.service.netty.StreamingObjectMapperHelper;
+import info.bitrich.xchangestream.service.pusher.PusherStreamingService;
+import io.reactivex.Observable;
+import org.knowm.xchange.currency.Currency;
+import org.knowm.xchange.dto.account.Balance;
+import org.knowm.xchange.dto.account.Wallet;
+
+import java.util.*;
+
+public class CoinmateStreamingAccountService implements StreamingAccountService {
+
+    private final PusherStreamingService service;
+    private final String userId;
+    private final Set<Wallet.WalletFeature> walletFeatures = new HashSet<>(Arrays.asList(Wallet.WalletFeature.TRADING, Wallet.WalletFeature.FUNDING));
+
+    public CoinmateStreamingAccountService(PusherStreamingService service, String userId) {
+        this.service = service;
+        this.userId = userId;
+    }
+
+    public Observable<Balance> getBalanceChanges(Currency currency, Object... args) {
+
+        return getCoinmateBalances().map(balanceMap -> balanceMap.get(currency.toString()))
+                .map((balance) -> {
+                    return new Balance(
```

Comment:
Use builder (constructors will likely be hidden in a future Xchange)

Reply:
Done